thoth/user_api/api_v1.py

```diff
@@ -855,11 +855,19 @@ def _get_workflow_status(parameters: dict, name_prefix: str, namespace: str):
     if not workflow_id.startswith(name_prefix):
         return {"error": "Wrong workflow id provided", "parameters": parameters}, 400
 
+
+def _get_status(node_name: str, analysis_id: str, namespace: str) -> typing.Tuple[typing.Dict[str, typing.Any], int]:
+    """Get status for a node in a workflow."""
+    result: typing.Dict[str, typing.Any] = {"parameters": {"analysis_id": analysis_id}}
     try:
-        status = _OPENSHIFT.get_workflow_status_report(workflow_id=workflow_id, namespace=namespace)
-    except OpenShiftNotFound:
-        return {"parameters": parameters, "error": f"Requested status for workflow {workflow_id!r} was not found"}, 404
-    return {"parameters": parameters, "status": status}, 200
+        status = _OPENSHIFT.get_workflow_node_status(node_name, analysis_id, namespace)
+    except OpenShiftNotFound as exc:
+        _LOGGER.exception(f"Status for {analysis_id} were not found: {str(exc)}")
```

Comment:
No logging needed here, it will produce unnecessary sentry reports.

Reply:
thanks, fixed